src/WebJobs.Script.WebHost/WebJobs.Script.WebHost.csproj

```diff
@@ -73,6 +72,7 @@
     <PackageReference Include="Microsoft.Azure.WebJobs.Logging" Version="4.0.2" />
     <PackageReference Include="Microsoft.Azure.WebSites.DataProtection" Version="2.1.91-alpha" />
     <PackageReference Include="Microsoft.VisualStudio.Web.CodeGeneration.Design" Version="2.2.3" />
+    <PackageReference Include="protobuf-net" Version="3.0.0" />
```

Comment:
Did one of the breaking change dependencies bring this in?

Reply:
That is right! `DotNetTI.BreakingChangeAnalysis` had dependency on this package. Added it back to ensure dependencies do not change